### PR TITLE
Task/remove GET /validations

### PR DESCRIPTION
**Pull request by stevenzeiler**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,55 +38,11 @@ To require basic http authentication set the following environment variables:
...
-List of latest daily reports, one report for each validator
-Response:
-* Array of Reports
+## API Documentation
 
-#####GET /reports/:validation_public_key
-List of historical daily reports for a given validator
-Response:
-* Array of Reports
+(https://api-staging.validators.ripple.com)[https://api-staging.validators.ripple.com]
```
Comment (wilsonianb) [inline README.md]: staging?

File: api/controllers/ValidationsController.js [hunk 2]
```diff
@@ -54,29 +54,6 @@ module.exports = {
...
-        res.status(200).json({ 
-          validations: validations 
-        }) 
-      })
-  },
-
-  /**
    * `ValidationsController.indexByLedger()`
    */
   indexByLedger: function (req, res) {
```
Comment (wilsonianb) [inline api/controllers/ValidationsController.js]: Do we still need this?

**Comment by wilsonianb** [comment]
I think `countByValidatorInLast24Hours` can also be removed from the `Validations` model

File: README.md (hunk 1 above)
Comment (stevenzeiler) [inline README.md]: For now... until we have an SSL certificate we can't view the documentation at api.validators.ripple.com

File: api/controllers/ValidationsController.js (hunk 2 above)
Comment (stevenzeiler) [inline api/controllers/ValidationsController.js]: It might be useful data to provide. I would rather document the API and then see how relevant it is.

**Comment by stevenzeiler** [comment]
Agreed, I will remove `countByValidatorInLast24Hours`

**Comment by wilsonianb** [comment]
LGTM :+1:

**Comment by stevenzeiler** [comment]
:+1:

**Pull request merged**